src/lib/fees/parameters/types.ts

```diff
@@ -8,4 +8,4 @@ export type FeeCurveParameters = {
   FEE_CURVE_FOX_DISCOUNT_DELAY_HOURS: number
 }
 
-export type ParameterModel = 'SWAPPER' | 'THORCHAIN_LP'
+export type ParameterModel = 'SWAPPER' | 'THORCHAIN_LP' | 'THORSWAP'
```

Comment:
It's not 100% what we want as THORSWAP is not a model, it's a little shortcut, we could make this more systemic and avoid this but the value is not so big for now as it's supposed to last 2 months, wdyt?